transformer/transformer_test.go

```diff
@@ -23,7 +23,7 @@ func TestTransform(t *testing.T) {
 			DatasetID: "",
 		},
```

Comment:
I would suggest testing that the new transform logic works as expected. this Item model can be extended with :

```
		PopulationType: "PopLbl1",
		Dimensions: []models.ESDimensions{
			{Name: "Dim1", Label: "Lbl1", RawLabel: "RawLbl1"},
		},
```

And `esDocument1` in `prepareESMockResponse` func could be extended with:
```
		PopulationType:  models.ESPopulationType{Name: "Pop1", Label: "PopLbl1"},
		Dimensions: []models.ESDimensions{
			{Name: "Dim1", Label: "Lbl1", RawLabel: "RawLbl1"},
		},
```